project/Build.scala

```diff
@@ -1189,7 +1171,8 @@ object Build {
             "org.webjars" % "jquery" % "1.10.2" / "jquery.js" % "test"
           )
       )
-  ).withScalaJSCompiler.dependsOn(library, jasmineTestFramework % "test")
+  ).withScalaJSCompiler.withScalaJSJUnitPlugin
+      .dependsOn(library, jUnitRuntime % "test")
```

Comment:
The style guide is not really precise for the sbt DSL. But you can see at `jUnitTestOutputsJS` that it should look like this:
```scala
  ).withScalaJSCompiler.withScalaJSJUnitPlugin.dependsOn(
      library, jUnitRuntime % "test"
  )
```